app/components/provider_interface/activity_log/application_form.rb

```diff
@@ -0,0 +1,30 @@
+module ProviderInterface
+  module ActivityLog
+    class ApplicationForm
+      attr_reader :event, :section
+      def initialize(event:)
+        @event = event
+        @section = ::ApplicationForm::ColumnSectionMapping.by_column(event.audit.audited_changes.keys.first)
+      end
+
+      def event_description
+        "#{section.humanize} edited"
```

Comment:
Can we move to a locale like the suggestion above? e.g `provider_interface.activity_log.application_form.section`